doc/lib_num_to_formatted_str.md

```diff
@@ -5,22 +5,20 @@
 ``lib_num_to_formatted_str.ks`` provides several functions for changing numbers (scalers) into strings with specified formats
 
 ### padding
+#### Description
+This function will return a string matching the format defined by the parameters.
 
-Args:
 
-  1. A number (scaler), num             The number be formatted
-  2. A number (scaler), leadingLength   The minimum number of digits to the left of the decimal point.
-  3. A number (scaler), trailingLength  The number of digits to the right of the decimal point.
-  4. A boolean,         leadingSpace    If True there will be a space at the front of the returned string when the number to be formatted is positive
-      * defaulted to True
-  5. A number (scaler), roundType       Sets if function will round, floor, or ceiling the number to be formatted. 0 = rounding, 1 = floor, 2 = ceiling, NOTE: if not one of the 3 expected values ceiling will be used
-      * defaulted to 0
+| parameter        | type    | default | description                                                                                                                                                                     |
```

Comment:
I used [markdowntable.com](http://markdowntable.com/) to format the tables for easier plain text viewing.
Of course GitHub side-by-side makes it look disgusting.